scripts/simple_raco_execution.py

```diff
@@ -0,0 +1,87 @@
+
+'''
+A notional raco-only script for executing a MyriaL query
```

Comment:
Not yet, but I agree that it still represents the plan. I'd like to get close to an API-compliant refactoring merged first.  We can delete it or rename it for now to avoid confusion?